### ISSUE TITLE
a11y prototype - nav with toast notification

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-next#119]
For one of the next GMS sessions, we'd like to test the roles "alert" and "status" in the context of a toast notification.

For this review, we'll need the following updates:

- update the [navigation as list prototype](https://rawgit.com/andresgalante/jenn/master/navbar-list.html) so that clicking specific menu items trigger the display of a toast notification. For example
  - Clicking Technology could display an error or warning toast
  - Clicking Entertainment could display an informational toast notification
- We'll use the toast notifications that are currently included on the [toast notification prototype page](https://rawgit.com/andresgalante/jenn/master/toast.html), but these will require updates to include the `role` attribute
  - toast notifications that are warning or error should have `role="alert"`
  - toast notifications that are success should have `role="status"` (also for informational, but that type isn't included in the test page)

A few helpful notes:

- the [role `alert`](https://www.w3.org/TR/wai-aria-1.1/#alert) implicitly has `aria-live` set to `assertive`
- the [role `status`](https://www.w3.org/TR/wai-aria-1.1/#status) implicitly has `aria-live` set to `polite`
- it might be worth testing that those attributes are actually working as expected, in case we need to explicitly set them
- the [toast notifications](https://rawgit.com/andresgalante/jenn/master/toast.html) that are currently implemented have slight variations in them, to help us figure out how to best structure the related elements in a toast. Should we pick a single variation to use for this `aria-live` test?

**Comment by seanforyou23** [comment]
Demo available at https://rawgit.com/andresgalante/jenn/master/single-notification.html

**Comment by jgiardino** [comment]
Some things we noticed when working on this:

- There seem to be more differences in how `role="alert"` behaves vs `role="status"` besides just aria-live being assertive or polite. For our purposes: 
  - `role="alert"` seems to be be best fit for toast notifications
  - `role="alertdialog" would be best for confirmation prompts
  - `role="status"` had different behaviors across screen readers and browsers. I'm not sure if there's anything in patternfly where this should be used.
- Even though `role="alert"` implicitly has `aria-live="assertive"`, I found that at least with testing in VO, it should be explicitly defined.
